homeassistant/components/luci/manifest.json

```diff
@@ -3,7 +3,8 @@
   "name": "Luci",
   "documentation": "https://www.home-assistant.io/components/luci",
   "requirements": [
-    "openwrt-luci-rpc==1.1.0"
+    "openwrt-luci-rpc==1.1.0",
+    "packaging==19.1"
```

Comment:
Shouldn't `openwrt-luci-rpc` depend on `packaging`? Why are we adding it here?

Reply:
Agreed. It should. But currently the luci component is broken without this.
Dependency can be removed once @fbradyirl fixed this upstream. I suggest this as temporary fix.